### DISCUSSION TITLE
Calendar Objet

### DISCUSSION THREAD

**Discussion by JimitD** [JoelBender/bacpypes#388] [Q&A]
Hi Joel, 

I am looking for creating schedule Object & calendar Object. Do you have any sample example to create calendar Object? 

Currently I am using LocalScheduleObject1 & LocalScheduleObjet2 for schedule Object.

**Comment by JoelBender** [comment]
I do not have an easy example, the **bacpypes.local.schedule** is a good starting point.  Maybe clone that into a **bacpypes.local.calendar** module... and associated sample applications and tests of course!  :smile:

**Reply by JimitD** [reply]
OK Joel, I will try it out
